osgar/lib/test_quaternion.py

```diff
@@ -33,5 +34,12 @@ def test_rotate_z90(self):
         self.assertAlmostEqual(expected[1], actual[1], places=6)
         self.assertAlmostEqual(expected[2], actual[2], places=6)
 
+    def test_from_axis_angle(self):
+        actual = quaternion.from_axis_angle((0,0,1), math.radians(90))
+        expected = [0, 0, 0.7071068, 0.7071068]
+        self.assertAlmostEqual(expected[0], actual[0], places=6)
+        self.assertAlmostEqual(expected[1], actual[1], places=6)
+        self.assertAlmostEqual(expected[2], actual[2], places=6)
```

Comment:
why not to check expected[3]?

Reply:
copy & paste error (not thinking straight)